### PR TITLE
Report blockheight + blockhash + block timestamp

### PR DESCRIPTION
**Pull request by typescribe**

### CODE REVIEW

File: explorer/src/__tests__/entity/TaskRun.test.ts [hunk 1]
```diff
@@ -33,7 +33,14 @@ describe('entity/taskRun', () => {
       [jr.id],
     )
 
-    const retrieved = await db.manager.findOne(JobRun, jr.id)
+    const retrieved = await db
```
Comment (rupurt) [inline explorer/src/__tests__/entity/TaskRun.test.ts]: There's a query in the `jobRuns#show` controller that looks similar to this. Could that be extracted into a function within `entity/JobRun` and reused here?

File: explorer/package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "migration:run": "ts-node ./src/bin/migrator.ts migrate",
     "migration:revert": "ts-node ./src/bin/migrator.ts revert",
     "test:migration:run": "cross-env TYPEORM_NAME=test ts-node ./src/bin/migrator.ts migrate",
+    "test:migration:revert": "cross-env TYPEORM_NAME=test ts-node ./src/bin/migrator.ts revert",
```
Comment (rupurt) [inline explorer/package.json]: 👍

File: explorer/src/__tests__/entity/TaskRun.test.ts (hunk 1 above)
Comment (typescribe) [reply explorer/src/__tests__/entity/TaskRun.test.ts]: yeah, didn't realise that there was code duplication with prod code. will do

**Comment by typescribe** [comment]
@rupurt I've extracted the logic out using https://typeorm.io/#/custom-repository

**Comment by rupurt** [comment]
@typescribe I like it. So it's essentially just wrapping the other repository. I feel like we should add some unit tests for it though.

I will apply the same pattern to the admin stories.

File: explorer/src/__tests__/repositories/JobRunRepository.test.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+let db: Connection
+
+beforeAll(async () => {
+  db = await getDb()
+})
+
+afterAll(async () => closeDbConnection())
+
+describe('JobRunRepository tests', () => {
```
Comment (rupurt) [inline explorer/src/__tests__/repositories/JobRunRepository.test.ts]: 👍

**Pull request merged**